internal/proto/controller/storage/kms/store/v1/data_key_version_destruction_job_run_allowed_table_name.proto

```diff
@@ -0,0 +1,12 @@
+syntax = "proto3";
+
+package controller.storage.kms.store.v1;
+
+option go_package = "github.com/hashicorp/boundary/internal/kms/store;store";
+
+message DataKeyVersionDestructionJobRunAllowedTableName {
```

Comment:
PK tag?

Reply:
This is a view, it doesn't have a primary key.